.github/workflows/build.yml

```diff
@@ -105,6 +119,10 @@ jobs:
           name: pluginVerifier-result
           path: ${{ github.workspace }}/build/reports/pluginVerifier
 
+      # Run Qodana inspections
+      - name: Qodana - Code Inspection
+        uses: JetBrains/qodana-action@v2022.3.4
```

Comment:
This takes a long time to run but appears to run both IntelliJ and plugin inspections, so maybe it is worth it overall.